Project-Lily/Project-Lily/DogListViewController.swift

```diff
@@ -132,35 +132,9 @@ extension DogListViewController: UITableViewDelegate {
 
 extension DogListViewController {
     
-    func dogURL() -> URL? {
-        let urlString = "https://private-52aac-breeds1.apiary-mock.com/breeds"
-        guard let url = URL(string: urlString) else {return nil}
-        return url
-    }
     
-    func createDogURLSession() {
-        let session = URLSession.shared
-        guard let url = dogURL() else { return }
-        let dataTask = session.dataTask(with: url) { (data, response, error) in
-            if let error = error {
-                print("Failure! \(error)")
-            } else if let httpResponse = response as? HTTPURLResponse,
-                httpResponse.statusCode == 200 {
-                if let data = data {
-                    self.dogDatabase.dogsArray = self.parse(data: data)!
-                    DispatchQueue.main.async {
-                        self.dogListTableView.reloadData()
-                    }
-                }
-                return
-            } else {
-                print("Failure! \(response!)")
-            }
-
-        }
-        
-        dataTask.resume()
-    }
+    
+    
```

Comment:
🍰 Keep an eye out for extra lines sneeking into your diff like these ones. They're relatively harmless, but can create some noise and distract from whatever point/points you're making in your PR.

Getting comfy with a version control GUI like Github Desktop, SourceTree, or whatever works for you, will help you catch some of these more easily 😄